### PR TITLE
Frontend: com_config view review

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Review frontend com_config views

### Testing Instructions

Install sample testing data and navigate all the com_config frontend views.

### Expected result

All still works

### Actual result

All views have cs errors ;)

### Documentation Changes Required

None

### CODE REVIEW

File: components/com_config/view/modules/tmpl/default.php [hunk 1]
```diff
@@ -34,44 +34,32 @@
 ");
 ?>
 
-<form
-	action="<?php echo JRoute::_('index.php?option=com_config'); ?>"
-	method="post" name="adminForm" id="modules-form"
-	class="form-validate">
-
+<form action="<?php echo JRoute::_('index.php?option=com_config'); ?>" method="post" name="adminForm" id="modules-form" class="form-validate">
```
Comment (brianteeman) [inline components/com_config/view/modules/tmpl/default.php]: this looks harder to read to me

Comment (zero-24) [reply components/com_config/view/modules/tmpl/default.php]: Like all other views ;)

Comment (brianteeman) [reply components/com_config/view/modules/tmpl/default.php]: maybe they are better :)

File: components/com_config/view/modules/tmpl/default.php [hunk 2]
```diff
@@ -108,21 +95,17 @@ class="form-validate">
...
+								</div>
+								<div class="controls">
+									<?php echo $this->form->getInput('published'); ?>
+								</div>
 							</div>
-							<div class="controls">
-								<?php echo $this->form->getInput('published'); ?>
-							</div>
-						</div>
 						<?php endif ?>
```
Comment (Quy) [inline components/com_config/view/modules/tmpl/default.php]: Add `;`

Comment (zero-24) [reply components/com_config/view/modules/tmpl/default.php]: fixed. Thanks

File: components/com_config/view/templates/tmpl/default.php [hunk 3]
```diff
@@ -41,20 +39,15 @@
 				</button>
 			</div>
 		</div>
-
 		<hr class="hr-condensed" />
-
 		<div id="page-site" class="tab-pane active">
 			<div class="row-fluid">
 				<?php // Get the menu parameters that are automatically set but may be modified.
```
Comment (Quy) [inline components/com_config/view/templates/tmpl/default.php]: Separate to 2 PHP statements.

Comment (zero-24) [reply components/com_config/view/templates/tmpl/default.php]: fixed thanks.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a27fe7c12759fa705ad8cc0dfe335d172e3b8e86<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17216">issues.joomla.org/tracker/joomla-cms/17216</a>.</sub>

File: components/com_config/view/config/tmpl/default.php [hunk 4]
```diff
@@ -41,21 +39,16 @@
 				</button>
```
Comment (Quy) [inline components/com_config/view/config/tmpl/default.php:41]: Line 33 and 38: multiple lines and add semicolon

File: components/com_config/view/modules/tmpl/default.php [hunk 5]
```diff
@@ -32,46 +32,34 @@
...
 						<span class="icon-apply"></span>
 						<?php echo JText::_('JAPPLY') ?>
 					</button>
 				</div>
 				<div class="btn-group">
-					<button type="button" class="btn"
-						onclick="Joomla.submitbutton('config.save.modules.save')">
+					<button type="button" class="btn" onclick="Joomla.submitbutton('config.save.modules.save')">
 						<span class="icon-save"></span>
 						<?php echo JText::_('JSAVE') ?>
```
Comment (Quy) [inline components/com_config/view/modules/tmpl/default.php]: Add semicolon

File: components/com_config/view/modules/tmpl/default.php [hunk 6]
```diff
@@ -32,46 +32,34 @@
...
 					</button>
 				</div>
 				<div class="btn-group">
-					<button type="button" class="btn"
-						onclick="Joomla.submitbutton('config.cancel.modules')">
+					<button type="button" class="btn" onclick="Joomla.submitbutton('config.cancel.modules')">
 						<span class="icon-cancel"></span>
 						<?php echo JText::_('JCANCEL') ?>
```
Comment (Quy) [inline components/com_config/view/modules/tmpl/default.php]: Add semicolon

File: components/com_config/view/modules/tmpl/default.php [hunk 7]
```diff
@@ -32,46 +32,34 @@
...
 		<!-- Begin Content -->
 		<div class="span12">
-
 			<div class="btn-toolbar" role="toolbar" aria-label="<?php echo JText::_('JTOOLBAR'); ?>">
 				<div class="btn-group">
-					<button type="button" class="btn btn-primary"
-						onclick="Joomla.submitbutton('config.save.modules.apply')">
+					<button type="button" class="btn btn-primary" onclick="Joomla.submitbutton('config.save.modules.apply')">
 						<span class="icon-apply"></span>
 						<?php echo JText::_('JAPPLY') ?>
```
Comment (Quy) [inline components/com_config/view/modules/tmpl/default.php]: Add semicolon

File: components/com_config/view/modules/tmpl/default.php [hunk 8]
```diff
@@ -32,46 +32,34 @@
...
 				</div>
 			</div>
-
 			<hr class="hr-condensed" />
-
 			<legend><?php echo JText::_('COM_CONFIG_MODULES_SETTINGS_TITLE'); ?></legend>
-
 			<div>
 				<?php echo JText::_('COM_CONFIG_MODULES_MODULE_NAME') ?>
 				<span class="label label-default"><?php echo $this->item['title'] ?></span>
```
Comment (Quy) [inline components/com_config/view/modules/tmpl/default.php]: Line 64-65: Add semicolon

File: components/com_config/view/modules/tmpl/default.php [hunk 9]
```diff
@@ -80,7 +68,6 @@ class="form-validate">
 				<span class="label label-default"><?php echo $this->item['module'] ?></span>
```
Comment (Quy) [inline components/com_config/view/modules/tmpl/default.php]: Line 67-68: add semicolon

File: components/com_config/view/templates/tmpl/default_options.php [hunk 10]
```diff
@@ -12,36 +12,24 @@
...
-	<?php echo $this->form->renderFieldset($name); ?>
-</fieldset>
-	<?php endforeach;
-	endif;
+<?php else : ?>
+	<?php // Fall-back to display all in params ?>
+	<?php foreach ($fieldSets as $name => $fieldSet) : ?>
+		<?php $label = !empty($fieldSet->label) ? $fieldSet->label : 'COM_CONFIG_' . $name . '_FIELDSET_LABEL'; ?>
+		<?php if (isset($fieldSet->description) && trim($fieldSet->description)) : ?>
+			<?php echo '<p class="tip">' . $this->escape(JText::_($fieldSet->description)) . '</p>'; ?>
```
Comment (Quy) [inline components/com_config/view/templates/tmpl/default_options.php]: Move `<p class="tip">` and `</p>` outside.

File: components/com_config/view/modules/tmpl/default_options.php [hunk 11]
```diff
@@ -9,47 +9,35 @@
...
-<?php echo JHtml::_('bootstrap.endSlide'); ?>
-<?php endforeach; ?>
+<?php $fieldSets = $this->form->getFieldsets('params'); ?>
+<?php echo JHtml::_('bootstrap.startAccordion', 'collapseTypes'); ?>
+	<?php $i = 0; ?>
+	<?php foreach ($fieldSets as $name => $fieldSet) : ?>
+		<?php $label = !empty($fieldSet->label) ? $fieldSet->label : 'COM_MODULES_' . $name . '_FIELDSET_LABEL'; ?>
+		<?php $class = isset($fieldSet->class) && !empty($fieldSet->class) ? $fieldSet->class : ''; ?>
+		<?php if (isset($fieldSet->description) && trim($fieldSet->description)) : ?>
+			<?php echo '<p class="tip">' . $this->escape(JText::_($fieldSet->description)) . '</p>'; ?>
```
Comment (Quy) [inline components/com_config/view/modules/tmpl/default_options.php]: Move `<p class="tip">` and `</p>` outside.

File: components/com_config/view/templates/tmpl/default.php [hunk 12]
```diff
@@ -41,20 +39,15 @@
 				</button>
```
Comment (Quy) [inline components/com_config/view/templates/tmpl/default.php:41]: Line 33 and 38: Multiple lines and add semicolon.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d3ba2a21feecb68c8ad3a4681cac401bb32c5ce1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17216">issues.joomla.org/tracker/joomla-cms/17216</a>.</sub>

**Pull request closed without merging**